### ISSUE TITLE
Provide visible keyboard focus

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aurora#260]
Keyboard focus is missing in:

- Aurora logo
- Left sidebar menu items
- Admin toggle
-  “All Transfers” and “My Transfers” tabs on Transfers page
- Previous - Next and page numbers for tables
- Table row links
- Rights statement table buttons in Transfer Detail page
- "Note" and "Details" buttons in Appraisal Queue
- "All Users" and "Users by Organization" tabs
- "Create Accession" page

**Comment by helrond** [comment]
Closed by #271
